src/Microsoft.AspNetCore.Rewrite/Internal/IISUrlRewrite/IISUrlRewriteRule.cs

```diff
@@ -13,16 +13,19 @@ public class IISUrlRewriteRule : IRule
         public UrlMatch InitialMatch { get; }
         public IList<Condition> Conditions { get; }
         public UrlAction Action { get; }
+        public bool Global { get; }
 
         public IISUrlRewriteRule(string name,
             UrlMatch initialMatch,
             IList<Condition> conditions,
-            UrlAction action)
+            UrlAction action,
+            bool? global = null)
```

Comment:
@davidpeden3 based on what you currently have they are in fact the same, perhaps I should have clarified this a little further. Consider the following:
```
public class Foo
{
    public bool Bar { get; }
    public Foo(bool bar = false) // My proposal
    {
        Bar = bar;
    }
}
```
Is cleaner as a tri-state `bool` is meaningless as `null` is always `false`.
```
public class Foo
{
    public bool Bar { get; }
    public Foo(bool? bar = null) // What you currently have
    {
        Bar = bar ?? false;
    }
}
```
Either way you can instantiate `Foo` like this:
```
var foo = new Foo(); // foo.Bar is false
var foo = new Foo(false); // foo.Bar is false
var foo = new Foo(true); // foo.Bar is true
```